src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -112,37 +116,40 @@ permissionStatus.onchange = () => {
 };
 ```
 
-## 🆕 The new image-focused portion of the Asynchronous Clipboard API {: #images }
+## The new image-focused portion of the Asynchronous Clipboard API {: #images }
 
-### Copy: Writing an image to the clipboard {: #copy-image }
+### Copy: writing an image to the clipboard {: #copy-image }
 
 The new `navigator.clipboard.write()` method can be used for copying images
 to the clipboard. Like [`writeText()`](#copy-text), it is asynchronous, and
 Promise-based. Actually, `writeText()` is just a convenience method for the
 generic `write()` method.
 
-In order to write an image to the clipboard, you need the image as a
-[`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob). One way to
-achieve this is by `fetch`ing (or `XMLHttpRequest`ing) the image from a
-server and getting the response body as a [Blob][blob] (or for XHR,
-by setting the [`responseType`][blog-response-type] to `'blob'`).
-Another method to `Blob`ify an image is to write the image to a canvas, then
-call the `canvas`’s [`toBlob()`][to-blob] method.
+To write an image to the clipboard, you need the image as a
+[`Blob`][blob]. One way to do
+this is by requesting the image from an server by calling `fetch()` (or
+`XMLHTTPReuest()`). The `response` object returned by `fetch()` has a [`blob()`
+method][blob-method] and
+`XMLHTTPRequest()` let's you set `"blob"` as the [`responseType`][blog-response-type]. 
+
+Calling the server may not be desireable or possible for a variety of reasons.
+Fortunately, you can also write the image to a canvas, then call
+[`HTMLCanvasElement.toBlog()`][to-blob].
```

Comment:
I reworked this whole paragraph to avoid using API keywords as different parts of speech. Changing API keywords also requires mixing fonts in the same word, which should also not be done.

Reply:
Thanks!